### PR TITLE
Add OpenStack Group content module

### PR DESCRIPTION
**Pull request by emonty**

### CODE REVIEW

File: cloud/openstack/os_user_group.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+requirements:
+    - "python >= 2.6"
+    - "shade"
+'''
+
+EXAMPLES = '''
+# Add the demo user to the demo group
+- os_user_group: user=demo group=demo
+'''
+
```
Comment (omgjlk) [inline cloud/openstack/os_user_group.py:56]: Output should be documented, well looks like none, so disregard.

**Comment by omgjlk** [comment]
This looks fine to me.

**Comment by gregdek** [comment]
Marking for inclusion per @j2sol.

**Comment by gregdek** [comment]
ping @ansible/core team

**Comment by jimi-c** [comment]
Merged, thanks!

**Pull request merged**